data/data/aws/vpc/vpc-private.tf

```diff
@@ -35,3 +35,13 @@ resource "aws_route_table_association" "private_routing" {
   route_table_id = "${aws_route_table.private_routes.*.id[count.index]}"
   subnet_id      = "${aws_subnet.private_subnet.*.id[count.index]}"
 }
+
+resource "aws_nat_gateway" "nat_gw" {
+  count         = "min(${local.new_az_count}, 3)"
+  allocation_id = "${aws_eip.nat_eip.*.id[count.index]}"
+  subnet_id     = "${aws_subnet.public_subnet.*.id[count.index]}"
```

Comment:
Why move these public-subnet resources into `vpc-private.tf`?

Reply:
because they are private subnet resources, not public. the move should happen irrespective of this PR

Reply:
> because they are private subnet resources, not public...

Ah, because while this resource lives in the public subnet, its purpose is to service instances in the private subnets.

> the move should happen irrespective of this PR

Personally I'd rather collapse `vpc-private.tf` and `vpc-public.tf` together.  The unified file would only be ~100 lines.  I'd work up a PR, but there are other PRs in flight touching this code and I don't think we could get this file-rename landed before the coming freeze.